pkg/controllers/allocation/binpacking/packable.go

```diff
@@ -164,16 +165,24 @@ func (p *Packable) validateArchitecture(schedule *scheduling.Schedule) error {
 	return nil
 }
 
-func (p *Packable) validateOperatingSystem(schedule *scheduling.Schedule) error {
-	if schedule.Requirements.OperatingSystems().Intersection(p.OperatingSystems()).Len() == 0 {
-		return fmt.Errorf("operating system %s is not in %v", p.OperatingSystems(), schedule.Requirements.OperatingSystems().List())
+func (p *Packable) validateZones(schedule *scheduling.Schedule) error {
+	packableZoneSet := sets.String{}
```

Comment:
consider s/packableZoneSet/zones. I try to avoid naming by variables after their type (`set`).